[2024-07-03T17:50:39.568+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T17:50:39.589+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_to_postgres2.load_csv_to_postgres manual__2024-07-03T17:50:37.942062+00:00 [queued]>
[2024-07-03T17:50:39.599+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_to_postgres2.load_csv_to_postgres manual__2024-07-03T17:50:37.942062+00:00 [queued]>
[2024-07-03T17:50:39.600+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-03T17:50:39.613+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_csv_to_postgres> on 2024-07-03 17:50:37.942062+00:00
[2024-07-03T17:50:39.627+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=246) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T17:50:39.630+0000] {standard_task_runner.py:63} INFO - Started process 248 to run task
[2024-07-03T17:50:39.631+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_to_postgres2', 'load_csv_to_postgres', 'manual__2024-07-03T17:50:37.942062+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/csv_to_postgres2.py', '--cfg-path', '/tmp/tmpo49oi_c4']
[2024-07-03T17:50:39.637+0000] {standard_task_runner.py:91} INFO - Job 68: Subtask load_csv_to_postgres
[2024-07-03T17:50:39.734+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_to_postgres2.load_csv_to_postgres manual__2024-07-03T17:50:37.942062+00:00 [running]> on host 9dffc1d7579f
[2024-07-03T17:50:39.849+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_to_postgres2' AIRFLOW_CTX_TASK_ID='load_csv_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-07-03T17:50:37.942062+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-03T17:50:37.942062+00:00'
[2024-07-03T17:50:39.850+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T17:50:39.875+0000] {base.py:84} INFO - Using connection ID 'postgres_destino' for task execution.
[2024-07-03T17:50:39.884+0000] {sql.py:487} INFO - Running statement: TRUNCATE TABLE categories, parameters: None
[2024-07-03T17:50:39.909+0000] {base.py:84} INFO - Using connection ID 'postgres_destino' for task execution.
[2024-07-03T17:50:39.924+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T17:50:39.925+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T17:50:39.936+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=csv_to_postgres2, task_id=load_csv_to_postgres, run_id=manual__2024-07-03T17:50:37.942062+00:00, execution_date=20240703T175037, start_date=20240703T175039, end_date=20240703T175039
[2024-07-03T17:50:39.978+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T17:50:39.997+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T17:50:40.001+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
